contrib/scripts/install_prereq

```diff
@@ -257,13 +310,25 @@ check_installed_zypper() {
 }
 
 handle_debian() {
-	if ! [ -x "$(command -v aptitude)" ]; then
-		apt-get install -y aptitude
-	fi
-	extra_packs=`check_installed_debs $PACKAGES_DEBIAN`
-	$testcmd aptitude update
+  ## Default for previous debian releases
+  extra_packs=`check_installed_debs $PACKAGES_DEBIAN`
+
+  ## Debian Bookworm
+  if [ $(cat /etc/debian_version | cut -d '.' -f 1) -eq 10 ]; then
+    extra_packs=`check_installed_debs $PACKAGES_DEBIAN_BUSTER`
+  fi
+
+  if [ $(cat /etc/debian_version | cut -d '.' -f 1) -eq 11 ]; then
+      extra_packs=`check_installed_debs $PACKAGES_DEBIAN_BULLSEYE`
+  fi
+
+  if [ $(cat /etc/debian_version | cut -d '.' -f 1) -eq 12 ]; then
+    extra_packs=`check_installed_debs $PACKAGES_DEBIAN_BOOKWORM`
+  fi
```

Comment:
Could this be simplified with a `case` statement? Something like:

```shell
case "$(cut -f1 -d. /etc/debian_version)" in
  10) extra_packs=$(check_installed_debs $PACKAGES_DEBIAN_BUSTER) ;;
  11) extra_packs=$(check_installed_debs $PACKAGES_DEBIAN_BULLSEYE) ;;
  12) extra_packs=$(check_installed_debs $PACKAGES_DEBIAN_BOOKWORM) ;;
  *)  extra_packs=$(check_installed_debs $PACKAGES_DEBIAN) ;;
esac
```

Reply:
That's pretty. I'll update the PR and test it to reflect that concept.